RetailCoder.VBE/UI/RubberduckUI.de.resx

```diff
@@ -1963,104 +1963,101 @@ Möchten sie die Einstellungen in Rubberduck importieren?</value>
   <data name="Preview" xml:space="preserve">
     <value>Vorschau</value>
   </data>
-  <data name="CodeExplorer_OpenDesigner">
+  <data name="CodeExplorer_OpenDesigner" xml:space="preserve">
     <value>Designer öffnen</value>
   </data>
-  <data name="SourceControl_InitializeRepositoryTitle">
+  <data name="SourceControl_InitializeRepositoryTitle" xml:space="preserve">
     <value>Neues lokales Repository erstellen.</value>
   </data>
-  <data name="EmptyUIRefreshMessage">
+  <data name="EmptyUIRefreshMessage" xml:space="preserve">
     <value>Um die Projekte in der VBE zu parsen und zu analysieren.</value>
   </data>
-  <data name="CodeExplorer_ExportAll">
+  <data name="CodeExplorer_ExportAll" xml:space="preserve">
     <value>Projekt exportieren...</value>
   </data>
-  <data name="ToolsMenu_ExportProject">
+  <data name="ToolsMenu_ExportProject" xml:space="preserve">
     <value>Aktives Projekt exportieren</value>
   </data>
-  <data name="ImportCommand_OpenDialog_Filter_VBFiles">
+  <data name="ImportCommand_OpenDialog_Filter_VBFiles" xml:space="preserve">
     <value>VB Dateien</value>
   </data>
-  <data name="GeneralSettings_EnableSourceControl">
-    <value>Versionsverwaltung aktivieren. Benötigt Neustart zur Anwendung.</value>
-  </data>
```

Comment:
why was this dropped?

Reply:
Dropped in initial PR, then readded. I'll reintroduce it.